util/ledger_util_test.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/algorand/go-algorand-sdk/encoding/json"
+	"github.com/algorand/go-algorand/data/bookkeeping"
+)
+
+func TestReadGenesis(t *testing.T) {
```

Comment:
Imo, it's easier to debug test failures when each test case is in a separate test instead of everything in a single test.